sources.go

```diff
@@ -58,6 +60,7 @@ func File(name string) *Pipe {
 // `find -type f`. If the path doesn't exist or can't be read, the pipe's error
 // status will be set.
 func FindFiles(path string) *Pipe {
+	path = strings.TrimRight(path, "\n")
```

Comment:
Perhaps what we're really saying is that it would be useful for `FindFiles` and friends to be _filters_ as well as sources, isn't it? So we could write, for example:

```go
script.Args().FindFiles().Concat().Stdout()
```

...the effect of which would be to concatenate all files found recursively in the supplied path.